assets/_scss/components/_disclaimer.scss

```diff
@@ -1,16 +1,33 @@
 .usa-disclaimer {
+  @include media($small-screen) {
+    font-size: $h5-font-size;
+  }
   background-color: $color-gray-lighter;
-  font-size: 1.5rem;
+  font-size: $h6-font-size;
   padding: {
     bottom: .5rem;
     top: .5rem;
   }
 }
 
+// This hides the stage note of the disclaimer in anything smaller than 690px
 .usa-disclaimer-stage {
-  @include media($medium-screen) {
+  @include media(690px) {
```

Comment:
@msecret this is the exact location where it breaks. Before it used the medium breakpoint and it broke to 2 lines. If we use the large it'll disappear too early. What breakpoint do you think we should use?